azureproject/settings.py

```diff
@@ -93,3 +93,67 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+if 'RUNNING_IN_PRODUCTION' in os.environ:
+    # Configure allowed host names that can be served and trusted origins for Azure Container Apps.
+    ALLOWED_HOSTS = ['.azurecontainerapps.io']
+    CSRF_TRUSTED_ORIGINS = ['https://*.azurecontainerapps.io']
+    DEBUG = False
+    DEBUG_PROPAGATE_EXCEPTIONS = True
+
+    # SECURITY WARNING: keep the secret key used in production secret!
+    # Use this py command to create secret 
+    # python -c 'import secrets; print(secrets.token_hex())'
+    SECRET_KEY = os.getenv('AZURE_SECRET_KEY')
+
+    STATICFILES_STORAGE = 'whitenoise.storage.CompressedStaticFilesStorage'
+
+    # Configure database connection for Azure PostgreSQL Flexible server instance.
+    # AZURE_POSTGRESQL_HOST is the full URL.
+    # AZURE_POSTGRESQL_USERNAME is just name without @server-name.
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql',
+            'NAME': os.environ['AZURE_POSTGRESQL_DATABASE'],
+            'HOST': os.environ['AZURE_POSTGRESQL_HOST'],
+            'USER': os.environ['AZURE_POSTGRESQL_USERNAME'],
+            'PASSWORD': os.environ['AZURE_POSTGRESQL_PASSWORD'], 
+        }
+    }
+else:
+    # SECURITY WARNING: don't run with debug turned on in production!
+    DEBUG = True
+
+    # Use this py command to create secret 
+    # python -c 'import secrets; print(secrets.token_hex())'
+    SECRET_KEY = os.getenv('LOCAL_SECRET_KEY')
+
+    # Don't use Whitenoise to avoid having to run collectstatic first.
+    STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.StaticFilesStorage'
+
+    ALLOWED_HOSTS = ['*']
```

Comment:
Is this needed? DEBUG=True has special handling for ALLOWED_HOSTS: https://docs.djangoproject.com/en/5.0/ref/settings/#allowed-hosts

Reply:
Thanks, it sure doesn't look like it. :). Just set it to [] on next push.